packages/scikit-image/meta.yaml

```diff
@@ -13,12 +13,11 @@ source:
 requirements:
   host:
     - numpy
-  run:
     - distutils
```

Comment:
```suggestion
```

I guess setting `distutils` in a host dependency has no effect so I think we can simply remove that.